docs/source/delta-type-widening.md

```diff
@@ -4,15 +4,35 @@ description: Learn about type widening in Delta.
 
 # Delta type widening
 
-.. note:: This feature is available in preview in <Delta> 3.2 and above.
+.. note:: This feature is available in preview in <Delta> 3.2 and above, and fully supported in <Delta> 4.0 and above.
```

Comment:
@allisonport-db I created https://github.com/delta-io/delta/pull/4696 to update the type widening documentation page. It supersedes all changes made here to this file.

Can you quickly do a pass and merge it?